Samples/office-add-in-commands/auto-open-task-pane/README.md

```diff
@@ -0,0 +1,171 @@
+---
+page_type: sample
+urlFragment: word-auto-open-with-document
+products:
+  - office-add-ins
+  - office-word
+  - office
+  - office-365
+languages:
+  - javascript
+extensions:
+  contentType: samples
+  technologies:
+    - Add-ins
+  createdDate: '05/03/2023 10:00:00 AM'
+description: "Configure a document to automatically open your Office Add-in's task pane when the document is opened."
+---
+
+# Auto-open a task pane with a document
+
+## Summary
+
+Some scenarios for your Office Add-in may require the task pane to automatically open with certain documents. This sample shows how to configure a document to automatically open your task pane when the document is opened by a user. The auto-open task pane feature requires the [AddInCommands 1.1](https://learn.microsoft.com/javascript/api/requirement-sets/common/add-in-commands-requirement-sets) requirement set.
```

Comment:
Do we need to mention the requirement set requirement, since it's only ruling out iPads (especially since the "Applies to" section doesn't list iPads)?